src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -145,14 +164,14 @@ public void reportIncludesMultipleDifferentItemsInTransit() {
     verifyLocation(firstItemJson);
     verifyRequestWithSecondPickupServicePoint(firstItemJson, requestDate1, requestExpirationDate1);
     verifyLoanInFirstServicePoint(firstItemJson, checkInDate1);
-    verifyLastCheckIn(firstItemJson, checkInDate1, SERVICE_POINT_NAME_1);
+    verifyLastCheckIn(firstItemJson, expectedCheckInDateTime, SERVICE_POINT_NAME_1);
 
     JsonObject secondItemJson = getRecordById(items, nod.getId()).get();
     verifyItem(secondItemJson, nod, secondServicePointId);
     verifyLocation(secondItemJson);
     verifyRequest(secondItemJson, requestDate2, requestExpirationDate2, REQUEST_PATRON_GROUP_DESCRIPTION, SERVICE_POINT_NAME_2);
     verifyLoanInFirstServicePoint(secondItemJson, checkInDate2);
-    verifyLastCheckIn(secondItemJson, checkInDate2, SERVICE_POINT_NAME_1);
+    verifyLastCheckIn(secondItemJson, expectedCheckInDateTime, SERVICE_POINT_NAME_1);
```

Comment:
We’re the check in date times meant to be deliberately different for these items?

Reply:
Because prior to these changes, different dates were saved to item's last checkin date. Now it's always the system time and it is the same because the time is 'frozen' in tests.

Reply:
My understanding is that the intention of this test is to demonstrate that different records can have different values in the report, meaning that it is intentional that the check in dates be different for each of the records.

Does that make sense? Does that fit with your understanding?

Reply:
I am not sure. I think, these tests were created because the last check in data is needed in the report. Also, those tests were written in the image of tests for the last check in. With the changes as of this PR, the dates can not be different in tests as the time is "frozen".

Reply:
> I am not sure. I think, these tests were created because the last check in data is needed in the report. 

I believe that the original version of `reportIncludesMultipleDifferentItemsInTransit` was written due to feedback I gave on a previous pull request. 

I tend to give feedback that any endpoint that can return multiple records needs an example of doing so where as much state as possible is different as possible. I do this because it is easy to accidentally mix up the state between records and this go unnoticed in the testing.

> I have doubts that there was an intention of demonstrating different dates. 

I think of the last check in information as another aspect of state that can differ.

> Also, those tests were written in the image of tests for the last check in.

If last check in information is presented for the item in the context of loans, then I'd want a test there that demonstrated the dates can be different (and are correctly assigned) too.

>  With the changes as of this PR, the dates can not be different in tests as the time is "frozen".

Can the time be frozen at multiple points in the test, allowing the various check ins to happen at different times?

Reply:
The tests have been changed to show a variety of dates in the last checkin.

Reply:
Thank you